README.md

```diff
@@ -54,3 +54,25 @@ Run the following to format all json files in the schemas directory:
 ```
 gulp format
 ````
+
+## Validating with ajv
+
+Also included is a node based version of the json schema validation which may be used during development to assist with
+debugging errors. This returns more errors than we'd currently like due to the way polymorphism works for each of our
+blocks.
+
+The node implementation depends on yarn. Once installed install ajv using:
+
+```
+yarn
+```
+
+Run the ajv based version of validator from within the ajv/ directory.
+
+```
+DEBUG=validator node app.js
```

Comment:
We instruct how to install the python version required using pyenv.  We should do the same for node with nvm.